практика 1

Конструкция try без catch или finally может быть использована только в
try-with-resources, когда ресурсы закрываются автоматически,
благодаря реализации интерфейса AutoCloseable. В таких случаях, если произойдёт исключение в
блоке try, оно будет брошено в вызывающий контекст.

практика 2

Да, конструкция try с только блоком finally допустима в Java. Это значит, что у нас есть блок try,
 но без блока catch, и только блок finally, который всегда будет выполнен,
 независимо от того, возникло ли исключение в блоке try или нет.


практика 3

Нельзя. В языке Java можно использовать только один блок finally
для каждого блока try. Это обусловлено тем, что
блок finally уже предназначен для выполнения кода,
который должен быть выполнен независимо от того,
было ли исключение в блоке try или нет.
Второй блок finally не имеет смысла,
 поскольку выполнение программы и так
 гарантированно перейдёт к следующему коду после
 первого блока finally.